client/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
```

Comment:
@maskowe May need approval to add these packages

Reply:
Got approval for the tinymce from Cami on 5/26/21 and the dompurify is recommended for security when rendering html dynamically in the editor.